jacobian_follower/include/jacobian_follower/jacobian_follower.hpp

```diff
@@ -196,4 +196,14 @@ class JacobianFollower {
       std::vector<std::string> const &link_names) const;
 
   std::vector<std::string> getLinkNames() const;
+
+  template <typename A, typename B>
+  void validateNamesAndPositions(const std::vector<A> &joint_names, const std::vector<B> &joint_positions) const {
```

Comment:
A bit misleading error message, as in one case this checks the length of joint_names, and in another this checks the batch dimension. 

Still probably enough to point the user to debug the correct variables though